addons/mail/static/src/thread/message_delete_dialog.js

```diff
@@ -0,0 +1,23 @@
+/** @odoo-module */
```

Comment:
can't we use the confirmation dialog instead? I think it can completely replace most simple dialog like this one (see confirmation_dialog.js in web/core)